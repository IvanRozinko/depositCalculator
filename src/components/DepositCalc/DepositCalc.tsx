import React from 'react'
import { Paper, Divider, Container } from '@material-ui/core'; 
import { Theme, makeStyles, createStyles } from '@material-ui/core/styles';
import DepositInputs from '../DeposinInputs/DepositInputs';
import DepositCharts from '../DepositCharts/DepositCharts';

const useStyles = makeStyles((theme: Theme) => 
  createStyles({
    paper: {
      display: 'flex',
      margin: '40px',
      padding: '40px 20px',
      flexGrow: 1,
      backgroundColor: 'gray',
      flexDirection: 'row',
      [theme.breakpoints.down('sm')]: {
        flexDirection: 'column',
        margin: '10px 0'
      }
    }
  })
);

function DepositCalc() {
  
  const styles = useStyles();

  return (
    <Container 
     maxWidth='lg'
    >
      <Paper
        variant='elevation'
        className={styles.paper}
      >
        <DepositInputs />
        <Divider orientation='vertical' flexItem />
        <DepositCharts />
      </Paper>
    </Container>
  )
}

export default DepositCalc
