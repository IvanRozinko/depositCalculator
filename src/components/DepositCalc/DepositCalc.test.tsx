import DepositCalc from "./DepositCalc";
import { shallow } from 'enzyme';

describe('<DepositCalc /> :', () => {
  const wrapper = shallow(
    <DepositCalc />
  )

  it('should renders correctly', () => {
    expect(wrapper).toMatchSnapshot()
  })
})