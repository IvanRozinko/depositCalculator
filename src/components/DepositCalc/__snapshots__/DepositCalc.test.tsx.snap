// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DepositCalc /> : should renders correctly 1`] = `
<Styled(MuiBox)
  className="makeStyles-container-1"
>
  <WithStyles(ForwardRef(Container))
    maxWidth="lg"
  >
    <WithStyles(ForwardRef(Paper))
      className="makeStyles-paper-2"
      variant="elevation"
    >
      <DepositInputs />
      <WithStyles(ForwardRef(Divider))
        flexItem={true}
        orientation="vertical"
      />
      <DepositCharts />
    </WithStyles(ForwardRef(Paper))>
  </WithStyles(ForwardRef(Container))>
</Styled(MuiBox)>
`;
